qdrant-landing/content/articles/new-recommendation-api.md

```diff
@@ -0,0 +1,336 @@
+points, and the distances between those points are the biggest. The deeper we go, the more points we have, and the distances 
+get closer. The graph is built in a way that the points are connected to their closest neighbours at every layer. 
+
+All the points from a particular layer are also in the layer below, so switching the search layer while staying in the same 
+location is possible. In the case of transport networks, the top layer would be the airline hubs, well-connected but with big 
+distances between the airports. Local airports, along with railways and buses, with higher density and smaller distances, make 
+up the middle layers. Lastly, our bottom layer consists of local means of transport, which is the densest and has the smallest 
+distances between the points.
+
+You don’t have to check all the possible connections when you travel. You select an intercontinental flight, then a local one, 
+and finally a bus or a taxi. All the decisions are made based on the distance between the points. 
+
+The search process in HNSW is also based on similarly traversing the graph. Start from the entry point in the top layer, find 
+its closest point and then use that point as the entry point into the next densest layer. This process repeats until we reach 
+the bottom layer. Visited points and distances to the original query vector are kept in memory. If none of the neighbours of 
+the current point is better than the best match, we can stop the traversal, as this is a local minimum. We start at the biggest 
+scale, and then gradually zoom in.
+
+In this oversimplified example, we assumed that the distance between the points is the only factor that matters. In reality, we 
+might want to consider other criteria, such as the ticket price, or avoid some specific locations due to certain restrictions. 
+That means, there are various strategies for choosing the best match, which is also true in the case of vector recommendations. 
+We can use different approaches to determine the path of traversing the HNSW graph by changing how we calculate the score of a 
+candidate point during traversal. The default behaviour is based on pure distance, but Qdrant 1.6 exposes two strategies for the 
+recommendation API.
+
+### Average vector
+
+The default strategy, called `average_vector` is the previous one, based on the average of positive and negative examples. It 
+simplifies the recommendations process and converts it into a single vector search. It supports both point IDs and vectors as 
+parameters. For example, you can get recommendations based on past interactions with existing points combined with query vector 
+embedding. Internally, that mechanism is based on the averages of positive and negative examples and was calculated with the 
+following formula:
+
+```
+average_vector = avg(positive_vectors) + ( avg(positive_vectors) - avg(negative_vectors) )
+```
```

Comment:
```suggestion
$$
\text{average vector} = \text{avg}(\text{positive vectors}) + \left( \text{avg}(\text{positive vectors}) - \text{avg}(\text{negative vectors}) \right)
$$
```

Should we use [MathJax](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions) here? 

More important: This is not an average vector directly? This is a centroid of positive vectors and then some distance addition to it based on the diff? It'd be helpful to understand what this diff addition is doing for us

Reply:
No, it's not. We have that described already, but in a different article: https://qdrant.tech/articles/vector-similarity-beyond-search/#vector-features-recommendations